### PR TITLE
Books list

### PR DESCRIPTION
**Pull request by carzoglioemanuel**
## Summary

- Books list added

## Screenshot

image

## Trello Card

https://trello.com/c/6w6cqZqs/30-listado-de-libros

### CODE REVIEW

File: books.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="books-container">
+      <div class="book-presentation-container">
+        <div class="book-presentation">
```
Comment (martinmona) [inline books.html]: i think these divs aren't necessary

File: books.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          <img class="book-cover-presentation" src="assets/book-cover.png" />
+          <h2 class="book-title-presentation">Título del libro</h2>
+          <p class="book-detail-presentation">Autor del libro</p>
```
Comment (martinmona) [inline books.html]: the span tag would look better

**Review by martinmona — CHANGES_REQUESTED** [review]

File: scss/variables/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+$badge-height: 120px;
+$badge-width: 95px;
+$book-cover-height: 365px;
+$book-cover-width: 260pxx;
+$book-cover-list-height: 200px;
+$book-cover-list-width: 142px;
+$book-container-height: 415px;
+$book-container-width: 900px;
+$book-detail-font-size: 20px;
```
Comment (martinmona) [inline scss/variables/_variables.scss:9]: Maybe you wanna use this values in another places, so the name would be more generic like _$regular-font-size_

Comment (martinmona) [reply scss/variables/_variables.scss:9]: Also, isn't necessary to put everything into variables :laughing:

**Review by nvescio — APPROVED** [review]

**Pull request merged**